rocon_python_comms/src/rocon_python_comms/connections.py

```diff
@@ -237,26 +256,28 @@ class ConnectionCache(object):
     """
     Caches all of the connections living in a ros master. Use the 'update'
     method to force a refresh of the basic information for every connection.
-    For detailed information, run 'generate_type_info' and
-    'generate_xmlrpc_info' on the specific connection (this is a more
-    expensive call and you don't want to do this for every connection).
     """
-    __slots__ = ['connections',
-                 '_lookup_node',
-                 '_get_system_state'
-                 ]
+    __slots__ = [
```

Comment:
Feel free to drop these slots if they serve no purpose. Should be adhering more strictly to having sphinx docs for class variables and defining them all (even if None) in the init.